include/openssl/ssl.h.gen

```diff
@@ -489,18 +489,7 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
  */
 # define SSL_MODE_ASYNC 0x00000100U
 
-/*
- * When using DTLS/SCTP, include the terminating zero in the label
- * used for computing the endpoint-pair shared secret. Required for
- * interoperability with implementations having this bug like these
- * older version of OpenSSL:
- * - OpenSSL 1.0.0 series
- * - OpenSSL 1.0.1 series
- * - OpenSSL 1.0.2 series
- * - OpenSSL 1.1.0 series
- * - OpenSSL 1.1.1 and 1.1.1a
- */
-# define SSL_MODE_DTLS_SCTP_LABEL_LENGTH_BUG 0x00000400U
+# define SSL_MODE_DTLS_SCTP_LABEL_LENGTH_BUG 0
```

Comment:
I don't believe anything ever used this. Codesearch only shows embedded openssl copies. But sure, keep it, it eats no grass, as we say in .ch.

Reply:
Cute phrase!